hadoop-common-project/hadoop-common/src/main/proto/RpcHeader.proto

```diff
@@ -91,6 +91,7 @@ message RpcRequestHeaderProto { // the header for the RpcRequest
   optional RPCTraceInfoProto traceInfo = 6; // tracing info
   optional RPCCallerContextProto callerContext = 7; // call context
   optional int64 stateId = 8; // The last seen Global State ID
+  optional bytes spanContext = 9; // OpenTracing SpanContext
```

Comment:
we should add the OT span context in the RPCTraceInfoProto

Reply:
I'm moving it to `RPCTraceInfoProto` instead.

Reply:
Done in b8c63f3f5d65e4d94caef98587d468bdbab78e5d.